### PR TITLE
Update What's New with Descriptive sanity variable names

### PR DESCRIPTION
**Pull request by zacjones93**
Updates what's new component to use the newly created resource collections that have a more descriptive name [based on this comment](https://github.com/eggheadio/egghead-next/pull/589#discussion_r620701892) on the previous PR.

No visual changes:
Build the portfolio you need to be a badass web developer   egghead io

### CODE REVIEW

File: src/pages/new/index.tsx [hunk 1]
```diff
@@ -10,12 +10,16 @@ import Card, {CardResource} from 'components/pages/home/card'
...
   const [jumbotron, secondPrimary, thirdPrimary] = primary.resources
-  const [firstSide, secondSide, thirdSide] = side.resources
+  const [
+    firstSecondaryResource,
+    secondSecondaryResource,
+    thirdSecondaryResource,
+  ] = secondary.resources
 
   return (
-    <section className="sm:-my-5 -my-3 p-5 mx-auto max-w-screen-xl">
```
Comment (zacjones93) [inline src/pages/new/index.tsx:18]: @vojtaholik noticed some awkward padding and this `p-5` was culprit

File: src/pages/new/index.tsx [hunk 2]
```diff
@@ -29,9 +33,18 @@ const WhatsNewPage: FunctionComponent<any> = ({resource}) => {
           <CardHorizontal className="w-full" resource={thirdPrimary} />
         </div>
         <div className="grid gap-4">
-          <CardHorizontal className="h-auto flex" resource={firstSide} />
-          <CardHorizontal className="w-full flex" resource={secondSide} />
-          <CardHorizontal className="w-full flex" resource={thirdSide} />
+          <CardHorizontal
+            className="h-auto flex"
+            resource={firstSecondaryResource}
```
Comment (zacjones93) [inline src/pages/new/index.tsx:38]: these variable names are a little awkward

File: src/pages/new/index.tsx [hunk 3]
```diff
@@ -147,7 +160,7 @@ const CourseFeatureCard = ({resource, className}: any) => {
 
 export const whatsNewQuery = groq`*[_type == 'resource' && slug.current == "whats-new"][0]{
   title,
-	'primary': resources[slug.current == 'primary-resources'][0]{
+	'primary': resources[slug.current == 'new-page-primary-resource-collection'][0]{
```
Comment (zacjones93) [inline src/pages/new/index.tsx:163]: new resource slug

File: src/pages/new/index.tsx [hunk 4]
```diff
@@ -166,7 +179,7 @@ export const whatsNewQuery = groq`*[_type == 'resource' && slug.current == "what
   		},
     }
   },
-	'side': resources[slug.current == 'side-resources'][0]{
+	'secondary': resources[slug.current == 'new-page-secondary-resource-collection'][0]{
```
Comment (zacjones93) [inline src/pages/new/index.tsx:182]: new resource slug

**Review by laurosilvacom — APPROVED** [review]

**Pull request merged**